tests/integration/contributors_test.py

```diff
@@ -332,3 +313,60 @@ def test_post_get_put_to_ensure_API_consitency(test_client):
     put_data = json.loads(put_resp.data)
 
     assert put_data["contributor"] == new_contrib
+
+
+def test_existing_contributors_after_put(test_client):
+    # get existing contributor "rt.tchoutchou"
+    get_contrib = test_client.get("/contributors/rt.tchoutchou")
+    original_contrib_chou = json.loads(get_contrib.data)["contributors"][0]
```

Comment:
Please ctrl+F `_chou` on this PR > `t`chou
```suggestion
    original_contrib_tchou = json.loads(get_contrib.data)["contributors"][0]
```
:wink: